tests/amr/data/field/copy_pack/stream_pack/CMakeLists.txt

```diff
@@ -10,8 +10,8 @@ set(SOURCES_INC
 
 set(SOURCES_CPP
   test_main.cpp
-#  test_stream_pack_centered_ex.cpp disabled cause it's b0rked
-#  test_stream_pack_centered_ey.cpp disabled cause it's b0rked
+  test_stream_pack_centered_ex.cpp disabled cause it's b0rked
```

Comment:
so we can put it back once you account for the diffs in my branch `fix`

Reply:
are you sure ?

I have failures after merging